### ISSUE TITLE
GLMakie's place vs Makie

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaPlots/Makie.jl#1341]
The docs say:
![image](https://user-images.githubusercontent.com/61620837/134663715-6d9ebc01-64d6-465a-887b-eb8ced0de91e.png)

But the GLMakie repo says:
![image](https://user-images.githubusercontent.com/61620837/134663541-0e524b56-38ce-488e-b8a2-2fffe021cf26.png)

I am personally confused - now that GLMakie is read-only, and apparently not part of the "life" ecosystem, should I only call `using Makie`? How do I change backend?

**Comment by 4aHxKzD** [comment]
GLMakie used to have a dedicated repo, but now all Makie related packages live in this Makie.jl repository. You can find GLMakie here now: https://github.com/JuliaPlots/Makie.jl/tree/master/GLMakie.
So yeah you still have to call `using GLMakie`, if you want to use interactive plotting.

**Comment by KronosTheLate** [comment]
But that would be using the "read-only" repo in my screenshot, right? If I call `using GLMakie`, a seperate module `GLMakie` will be attempted loaded I am pretty sure - the compiler has no way of knowing that I want to use the module within `Makie`, right? So should I then call `using Makie.GLMakie`?

**Comment by 4aHxKzD** [comment]
This `git` repository is a mono repo for various Makie packages that make up the Makie ecosystem. If you enter
[GLMakie](https://github.com/JuliaPlots/Makie.jl/tree/master/GLMakie) or [CairoMakie](https://github.com/JuliaPlots/Makie.jl/tree/master/CairoMakie) you can see that both have a `Project.toml` file. They are separate packages that happen to live in a single git(hub) repository.
When you install any of these two packages `Pkg` knows that the code should be taken from this folders in this git repo, because someone registered them in this way.

TLDR, nothing changes for the user, just keep using the packages as if they have separate git(hub) repositories.

```
]add GLMakie
using GLMakie
```

**Comment by 4aHxKzD** [comment]
I don't know how to check `Pkg` registries but if you search `GLMakie` package on JuliaHub [here](https://juliahub.com/ui/Packages/GLMakie/nfnZR/0.4.6 ), you can see that its repo is this one (https://github.com/JuliaPlots/Makie.jl)

**Comment by SimonDanisch** [comment]
> the compiler has no way of knowing that I want to use the module within Makie, right 

This isn't the job of the compiler, but of the package manager - which uses the General registry to figure out where the code for a package resides.
